File: src/main/java/com/cargosystem/model/Cargo.java
package com.cargosystem.model;

import javax.persistence.*;
import java.math.BigDecimal;
import java.util.Set;

@Entity
@Table(name = "cargo")
public class Cargo {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String name;

    private BigDecimal price;

    @ManyToOne
    private Company company;

    public Cargo() {
    }

}

File: src/main/webapp/frontend-angular/src/app/_models/Cargo.ts
export class Cargo {
  id: number;
  name: string;
  price: number;
  companyName: string;

  constructor(id: number, name: string, price: number, companyName: string) {
    this.id = id;
    this.name = name;
    this.price = price;
    this.companyName = companyName;
  }
}
